treeherder/intermittents_commenter/commenter.py

```diff
@@ -120,6 +121,9 @@ def generate_bug_changes(self, startday, endday, alt_startday, alt_endday):
                     change_whiteboard += "[stockwell disable-recommended]"
             comment = template.render(
                 bug_id=bug_id,
+                total=counts.total,
+                failure_rate=round(counts.total / float(test_run_count), 3),
+                test_run_count=test_run_count,
```

Comment:
I would like to remove the failure_rate and test_run_count.  Those values are not very useful.